src/main/java/com/mael/payroll/paymentSchedules/MonthlySchedule.java

```diff
@@ -0,0 +1,4 @@
+package com.mael.payroll.paymentSchedules;
+
+public class MonthlySchedule implements PaymentSchedule {
```

Comment:
Yeah, they do, Felipe was also shocked that I had an interface with no methods, but they do get used later. It's just the way the book does things. I'm gonna try to do things my way from now on.